better-font-awesome.php

```diff
@@ -513,18 +543,67 @@ public function text_callback( $args ) {
      * @return  string  Usage text.
      */
     public function get_usage_text() {
```

Comment:
I guess we're going off on a bit of a tangent here...

When I've done something like this - I've added an args param that is an array of values that can be used in the template.

This is kinda similar to how you would use a templating like mustache. I think Mustache you pass JSON when rendering a template.

Probably over complicating things now!